news/2021-12-27-clickers-choice-2021.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "The Best osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+It's time for the community to decide the top 20 osu! players of 2021!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo-play and tournaments. Now that the year's coming to an end, we're bringing you the **9th iteration of the international top 20 voting**, courtesy of my latest project: [Clicker's Choice](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage)!
+
+The Top 20 vote will be run on the Clicker's Choice website, which is already home to the top player voting for Germany, Poland, Canada, UK and many other countries. Clicker's Choice aims to make it easier for users to vote, while also making it simpler for the host to tabulate and evaluate results.
+
+As a voter, it's pretty straightforward - just log in through osu! and start voting! **Everything regarding the criteria and deadlines are on the voting page as well.** Here's a showcase of the entire process:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to join the hwchr projects discord server or contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
+
+[TOP 20 VOTING](https://choice.hwc.hr/vote/6?mtm_campaign=news-post-2021&mtm_kwd=top-20-voting) | [HOMEPAGE](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage) | [DISCORD](https://u.hwc.hr/discord))
```

Comment:
```suggestion
[TOP 20 VOTING](https://choice.hwc.hr/vote/6?mtm_campaign=news-post-2021&mtm_kwd=top-20-voting) | [HOMEPAGE](https://choice.hwc.hr/?mtm_campaign=news-post-2021&mtm_kwd=homepage) | [DISCORD](https://u.hwc.hr/discord)
```

Reply:
Change the web link to `https://choice.hwc.hr/vote/6` and `https://choice.hwc.hr` ?

Reply:
Any chance I can leave them as-is? I would like to know how many people come from this news post by including a campaign name

Reply:
Fixed the dangling parenthesis 👍

Reply:
Remove discord link and remove `mtm_kwd` whatever that is.